java/core/src/test/org/apache/orc/TestNewIntegerEncoding.java

```diff
@@ -335,9 +335,9 @@ public void testBasicDelta4(OrcFile.EncodingStrategy encodingStrategy) throws Ex
   public void testDeltaOverflow() throws Exception {
     TypeDescription schema = TypeDescription.createLong();
 
-    long[] inp = new long[]{4513343538618202719l, 4513343538618202711l,
-        2911390882471569739l,
-        -9181829309989854913l};
+    long[] inp = new long[]{4513343538618202719L, 4513343538618202711L,
+        2911390882471569739L,
+        -9181829309989854913L};
```

Comment:
Shall we have this block like this?  I find it more easy on my eyes. We don't have to fix these things now. I can take care of this later

```
long[] inp = new long[]{4513343538618202719L, 4513343538618202711L,
                        2911390882471569739L, -9181829309989854913L};                                                                               
```

Reply:
May I asked which rule complains about that, @mystic-lama ?

As you see, this PR aims to introduce the minimal changes which means to touch only `Checkstyle` really complains. The verification is described in the PR description, too.

To simply put, if we mix different themes arbitrarily like that, it tends to make a mistake because of the human error factor. It makes reviewers difficulty to review the PRs. It ends up as a situation we cannot merge it.

Of course, we can take care of it later independently.

Reply:
I agree with your point. I don't recall the rule, and did not mean to fix it in this PR
This is more for future, when we make code changes. I shall keep an eye for this and next time it hits, may be we can refine the rule.